SIL.WritingSystems/LanguageLookup.cs

```diff
@@ -258,6 +251,24 @@ private bool AddLanguage(string code, string threelettercode, string full = null
 			return true;
 		}
 
+		private static string RegionToCountry(string region)
+		{
+			string country;
+			if (StandardSubtags.IsPrivateUseRegionCode(region))
+			{
+				country = region == "XK" ? "Kosovo" : "Unknown private use";
```

Comment:
You might also consider leveraging cldr data here/ there are about 8 such codes cldr uses.

Reply:
@srl295 Would you be up for opening an issue for that with a pointer to the relevant CLDR data?